v2/src/theme.ts

```diff
@@ -7,12 +7,17 @@ const theme = createTheme({
   },
   palette: {
     primary: {
-      // main: '#60594D', // Umber
-      main: '#000000DE',
-    },
-    background: {
-      default: '#EBA937',
+      main: '#eba937',
+      contrastText: '#1E1608',
+      light: '#ffda68',
+      dark: '#b47a00',
     },
+    secondary: {
+      main: '#201613',
```

Comment:
Suggestion here, I think the back used by MUI for dark text looks pretty well with the honey colour. It's between black and dark grey.
Hex: #000000de
rgba(0, 0, 0, 0.87)

Reply:
It's used for the black colour in the website currently.
<img width="361" alt="image" src="https://user-images.githubusercontent.com/40524083/162663258-86492585-f586-4d30-aa8f-84f19c92d3c5.png">

Reply:
is this hex a valid hex? #000000de? I thought it would only be #000000

Reply:
Hex colors can take 3, 6 or 8 bytes. It's recognised by CSS.

Reply:
It's RGBA, where A is for opacity, so the last 2 bytes.